plugins/vpp/ifplugin/vppcalls/dump_interface_vppcalls.go

```diff
@@ -153,14 +155,19 @@ func (h *IfVppHandler) DumpInterfaces() (map[uint32]*InterfaceDetails, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to dump unnumbered interfaces: %v", err)
 	}
-	// Get vrf for every interface and fill DHCP if set
+	// Get interface VRF for every IP family, fill DHCP if set and resolve unnumbered interface setup
 	for _, ifData := range ifs {
 		// VRF
 		vrf, err := h.GetInterfaceVrf(ifData.Meta.SwIfIndex)
 		if err != nil {
 			return nil, fmt.Errorf("interface dump: failed to get VRF from interface %d: %v", ifData.Meta.SwIfIndex, err)
 		}
-		ifData.Interface.Vrf = vrf
```

Comment:
ifData.Interface.Vrf is no longer filled by DumpInterfaces?

Reply:
Or if ipv6Vrf and ipv4Vrf can be different, maybe Interface model should be extended to contain both?

Reply:
I think we should stick with single VRF definition in NB and let the vppcalls create both in VPP.

Reply:
Creating the one needed at the time should be enough. We can store both VRFs in the interface metadata and put the correct one to the ifData.Interface.Vrf, so the value in the dumped data will match the NB and we do not need any extra fields in the model.